app/scripts/proactive/model/Job.js

```diff
@@ -30,7 +30,7 @@ define(
                 "Property Name": { validators: ['required'], fieldAttrs: {'placeholder': '@attributes->name'} },
                 "Property Value": { validators: ['required'], fieldAttrs: {'placeholder': '@attributes->value'} }
             }},
-            "User Space Url": {type: "Text", fieldAttrs: {"data-tab": "Data Management", 'data-tab-help': 'Definition of dataspaces URLs', 'placeholder': 'userSpace->@attributes->url', "data-help":"A User Space which is a personal user data storage. Usually you set this url if you want to use your own dataspace server, not the one that is included into the Scheduler."}},
+            "User Space Url": {type: "Text", fieldAttrs: {"data-tab": "Data Management", 'data-tab-help': 'DataSpaces allow to automatically manage File transfer at the level of Job and Tasks.', 'placeholder': 'userSpace->@attributes->url', "data-help":"A User Space which is a personal user data storage. Usually you set this url if you want to use your own dataspace server, not the one that is included into the Scheduler."}},
```

Comment:
I would suggest : "DataSpaces allow to automatically manage File transfer at the level of Job and Tasks. In this section you can override DataSpaces provided automatically by the scheduler and enter your custom DataSpaces urls"